GitVersionConfig.yaml

```diff
@@ -3,3 +3,5 @@ next-version: 6.0
 branches:
   develop:
     tag: alpha
+  releases?[/-]:
```

Comment:
@SimonCropp @Particular/nservicebus-maintainers I think this is a resonable default going forwards => will make minor releases be "RC" status right away?

This way you only have to change this when going into a new major (change it down to `beta`)

@JakeGinnivan is there a way to configure GV to say "Major releases should start with `beta` but minor/patch should start with `RC`" ?